server/models/user.coffee

```diff
@@ -20,3 +20,14 @@ module.exports.getDisplayName = (callback) ->
             callback null, name
         else
             callback null, null
+
+
+module.exports.getEmail = (callback) ->
+    cozydb.api.getCozyUser (err, user) ->
+        if user?
+            callback null, user.email
+        else
+            # FIXME: Why the hell do we returns err == null if we cannot get a
```

Comment:
I'm totally satisfied by the CozyDB API, I just wanted to keep attention to the fact that, if we cannot get the user (which can be considered as an error in this case) we call the callback but with its `err` argument to `null` (which isn't obvious because if the callback test the `err` argument to detect an error, it'll consider the response as valid).

I decided to return `{email: undefined}` in `err` in this case, but I don't really know if it's really a good architecture choice (because we don't do that before at L22). This is the reason of my comment, but you're true, I should probably don't let it in code comments :wink:.

BTW, what do we do in this case (error raising)?